script.js

```diff
@@ -0,0 +1,137 @@
+const userInput = document.querySelector(".s-text-input");
+const searchBtn = document.querySelector(".s-btn");
+const ul = document.querySelector("ul");
+const div = document.querySelector("div");
+let isPrev = false;
+
+
+
+function deletePrevList() {
+  const allLinks = document.querySelectorAll("li");
+  ul.innerHTML = "";
+}
+
+function progressBar(width = 0) {
+  let elem = document.querySelector(".loader-bar");
+  elem.classList.add("move");
+  let id = setInterval(frame, 1);
+  function frame() {
+    if (width >= 70) {
+      clearInterval(id);
+    } else {
+      width++;
+      elem.style.width = width + "%";
+    }
+  }
+}
+function removeProgressBar(width = 70) {
+  let elem = document.querySelector(".loader-bar");
+  elem.classList.add("move");
+  let id = setInterval(frame, 1);
+  function frame() {
+    if (width >= 100) {
+      clearInterval(id);
+    } else {
+      width++;
+      elem.style.width = width + "%";
+    }
+  }
+  setTimeout(() => {
+    elem.classList.remove("move");
+  }, 300);
+}
+
+function displayList(data, logos, i = 0) {
+  console.log(data);
+  data.forEach((element) => {
+    if (i < 10) {
+      console.log(element);
+      const li = document.createElement("li");
+      const a = document.createElement("a");
+      const img = document.createElement('img');
+      img.classList.add('logo');
+      img.src = logos.src[i];
+      a.href = `company.html?symbol=${element.symbol}`;
+      a.textContent = element.name;
+      a.textContent += `(${element.symbol})`;
+      a.textContent += '  ';
+      a.textContent += `(${logos.percent[i]}%)`;
+      li.appendChild(img);
+      li.appendChild(a);
+      ul.appendChild(li);
+      i++;
+    }
+  });
+  isPrev = true;
+}
+
+async function getSearchResults(userInput) {
+  try {
+    const response = await fetch(
+      `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=${userInput}&amp;limit=10&amp;exchange=NASDAQ`
```

Comment:
use baseUrl variable to store the adress of the API
https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/